e2etest/zt_basic_copy_sync_remove_test.go

```diff
@@ -41,7 +41,7 @@ import (
 
 // ================================  Copy And Sync: Upload, Download, and S2S  =========================================
 func TestBasic_CopyUploadSingleBlob(t *testing.T) {
-	RunScenarios(t, eOperation.CopyAndSync(), eTestFromTo.AllUploads(), eValidate.AutoPlusContent(), anonymousAuthOnly, allCredentialTypes, params{
+	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.LocalBlobFS()), eValidate.AutoPlusContent(), anonymousAuthOnly, anonymousAuthOnly, params{
```

Comment:
Would it be possible to add LocalBlobFS to AllUploads? What about testing LocalFile?

Reply:
Coincidentally, I think this was another accident. I believe I'd just reduced the test conditions to test exactly what I was working on.

Reply:
LocalBlobFS was present